javascript/ql/src/semmle/javascript/dataflow/Configuration.qll

```diff
@@ -788,7 +878,12 @@ private predicate reachableFromStoreBase(
   or
   exists(DataFlow::Node mid, PathSummary oldSummary, PathSummary newSummary |
     reachableFromStoreBase(prop, rhs, mid, cfg, oldSummary) and
-    flowStep(mid, cfg, nd, newSummary) and
+    (
+      flowStep(mid, cfg, nd, newSummary) 
+      or
+      existsCopyProperty(mid, nd, prop, cfg) and
```

Comment:
Have you tried using `isAdditionalCopyPropertyStep` directly here? I don't see why we need the recursive helper predicate at all.

Reply:
You are right, with some of my previous implementation the recursive helper was needed, but that is not the case anymore.